website/docs/reference/resource-configs/delimiter.md

```diff
@@ -0,0 +1,76 @@
+---
+resource_types: [seeds]
+datatype: <string>
+default_value: ","
+---
+
+## Definition
+
+An optional seed configuration used to separate values in a [seed](/docs/build/seeds) with a string you provide. The delimiter defaults to comma.
+
+
+## Usage
+Specify delimiter in your `dbt_project.yml` file to globally separate all seed values:
+
+<File name='dbt_project.yml'>
+
+```yml
+seeds:
+  +quote_columns: False
+  +delimiter: ","
+```
+
+</File>
+
+
+Or only separate values with a delimiter in the seeds/mappings directory:
+<File name='seeds/properties.yml'>
+
+
+```yml
+version: 2
+
+seeds:
+  quote_columns: False
+  delimiter: ","
+```
+
+</File>
+
+## Examples
+For a project with:
+
+* `name: jaffle_shop` in the `dbt_project.yml` file
+* `seed-paths: ["seeds"]` in the `dbt_project.yml` file
+
+### Use a comma delimiter
```

Comment:
This is the default, you wouldn't need to explicitly set this unless you'd overridden the global delimiter ```yml
seeds:
  jaffle_shop: 
    +delimiter: "|"
    mappings:
      +delimiter: ","
```